### ISSUE TITLE
Add method Ping 

### ISSUE THREAD

**Issue by andruwa13** [CastawayLabs/cachet-monitor#106]
Please add method ping

**Comment by thyspir** [comment]
Yes, please add this feature so that we can monitor our internal IP

**Comment by matejkramny** [comment]
Hey, we looked into ICMP test. It wasn't really readily available in Go.

Anyone know a good way to do it?

Btw checkout the v3 branch.

Matej
